targets/TARGET_STM/TARGET_STM32F0/lp_ticker_defines.h

```diff
@@ -0,0 +1,28 @@
+/* mbed Microcontroller Library
+ * SPDX-License-Identifier: BSD-3-Clause
+ ******************************************************************************
+ *
+ * Copyright (c) 2015-2020 STMicroelectronics.
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+ */
+
+#ifndef __LP_TICKER_DEFINES_H
```

Comment:
Given that lp_ticker.c is common for all STM families, can't this file be common for all families too, like us_ticker_defines.h?

Would just need the extra ifdefs for `DEVICE_LPTICKER` and `MBED_CONF_TARGET_LPTICKER_LPTIM` you see in the C file.